components/DatePicker.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react'; // React
+import React, { useRef } from 'react'; // React
```

Comment:
I think Anish has some changes coming in to remove import React, so don't worry about it here.